static/app/components/search/searchResult.tsx

```diff
@@ -38,7 +38,7 @@ function renderResultType({resultType, model}: Result['item']) {
 }
 
 function SearchResult({item, matches, highlighted}: Props) {
-  const params = useParams<{orgId: string}>();
+  const params = useParams();
```

Comment:
Why was the `orgId` type specialization removed here?

Reply:
I can put it back. I had initially removed the call to `useParams()` in my quest to remove routing parameters, but that failed. When I restored the call to `useParams()` I forgot to include the type annotation.